api/routes.py

```diff
@@ -48,6 +48,15 @@ def get_locale():
     return request.accept_languages.best_match(languages)
 
 
+@app.after_request
+def print_cache(response):
+    if hasattr(app, "_db") and HasFullTableCache.CACHE_ATTRIBUTE in app._db.info:
```

Comment:
Can it add additional overhead? Can we add an additional check to run it only if `DEBUG` level was turned on?